docs/components/muc-button.md

```diff
@@ -18,9 +21,10 @@ import Basic from './demo/MucButton/Basic.vue'
 
 ### Properties
 
-| Name | Type   | Default | Description    |
-| ---- | ------ | ------- | -------------- |
-| msg  | string | null    | Messge to show |
+| Name     | Type    | Default | Description                                                                                                                  |
+|----------|---------|---------|------------------------------------------------------------------------------------------------------------------------------|
+| variant  | string  | primary | The variant prop gives you easy access to several different button styles. Available are `primary`, `secondary` and `ghost`. |
+| disabled | boolean | false   | Let`s you indicate that the button is not currently interactive or clickable.                                                |
```

Comment:
Can you add the doc for the icon-Property, too?

Reply:
forgot the docs ... ☹

Reply:
added docs for icon and for the slot